ua/lviv/iot/algo/part1/lab1/Chair.java

```diff
@@ -0,0 +1,57 @@
+package ua.lviv.iot.algo.part1.lab1;
+import lombok.*;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter
+@Getter
+@ToString
+public class Chair {
+    private int id = 1;
+    private String material;
+    private int maxWeight;
+    private String owner;
+    private static Chair defaultChair = new Chair();
+  
+    public static Chair getInstance(){
```

Comment:
```suggestion
    public static Chair getInstance() {
```
Reformat this file, there are similar formatting issues here